site/app/templates/officeHoursQueue/SearchStudentQueue.twig

```diff
@@ -0,0 +1,117 @@
+<div id='student_queue_history'>
+    <br>
+    <h2>Student's Queue History</h2>
+    <p>Student ID</p>
+    <div id='search-student-queue'>
+        <input id='search-student-queue-input' data-testid='search-student-queue-input' type='text' placeholder="Enter student's name">
```

Comment:
Is there better placeholder text we can put here?

Reply:
Barb said it was fine.